### ISSUE TITLE
Add to the documentation how to update miniforge

### ISSUE THREAD

**Issue by arturdaraujo** [conda-forge/conda-forge.github.io#2367]
### Conda-forge documentation

- [x] I could not solve my problem using the conda-forge documentation.

### Installed packages

```shell
-
```

### Environment info

```shell
-
```

### Issue

I found it confusing to update miniforge, not sure if I should remove it and then install fresh. It was suggested that I should mention that in here for an update on the documentation.

https://github.com/conda-forge/miniforge/issues/674#event-15241234958

**Comment by arturdaraujo** [comment]
Ok, thanks

**Comment by hmaarrfk** [comment]
i don't think it is a bad idea to cross reference this, i'm just trying to think of a "scalable" way to do this...
